### PR TITLE
Navbar is adjustment for small screen

### PR DESCRIPTION
**Pull request by madhusudhan1234**
### Summary of the changes

In mobile devices in navigation in the initial state there was cancel icon was shown instead of hamburger icon so that is fixed by removing is-active class. Hovering the navbar-item provides the default bulma's primary color that is now changed to ruby logo red color. 

### Why need this

* Navigation should be good in mobile devices

### Main Changes

- [x] In small screen hamburger icon is now appear except cancel icon
- [x] In mobile devices hover color is changed to red color

### CODE REVIEW

File: _sass/_includes/_nav.scss [hunk 1]
```diff
@@ -27,3 +27,10 @@
 .rbnw-nav {
   box-shadow: 1px 0 2px 2px $navbar-box-shadow-color;
 }
+
+.navbar-link.is-active,
+.navbar-link:hover,
```
[the remark this answers is not in the document]
Comment (madhusudhan1234) [reply _sass/_includes/_nav.scss]: @houndci-bot  Finally Fixed !

File: _includes/home-nav.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="navbar home-nav">
   <div class="navbar-brand">
-    <div class="navbar-burger burger is-active" data-target="navbarExampleTransparentExample">
+    <div class="navbar-burger burger is-horizontal-center" data-target="navbarExampleTransparentExample">
```
Comment (madhusudhan1234) [inline _includes/home-nav.html]: will remove is-horizontal-center

File: _sass/_includes/_nav.scss [hunk 3]
```diff
@@ -9,6 +9,15 @@
   }
 }
 
+a {
+  &.navbar-item:hover {
```
[the remark this answers is not in the document]
Comment (madhusudhan1234) [reply _sass/_includes/_nav.scss]: what do you mean ?

**Comment by zoras** [comment]
@madhusudhan1234 the footer is not displayed in middle on mobile, can you fix that as well?

File: _sass/_includes/_nav.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (zoras) [reply _sass/_includes/_nav.scss]: it means `:hover` should be nested as well

**Pull request merged**